frontend/lib/components/drawer.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+import 'package:frontend/pages/home.dart';
+import 'package:frontend/pages/calendar.dart';
+import 'package:frontend/pages/user_events.dart';
+
```

Comment:
Be sure to leave a comment here to briefly describe this component's functionality